### DISCUSSION TITLE
v0.3.15

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#347] [Announcements]
## What's Changed
* Add drug keywords, craftable spell names, hypno triggers, and configu… by @littlesera in https://github.com/littlesera/LSCG/pull/344
* use more appropriate hook location to adjust height ratios by @littlesera in https://github.com/littlesera/LSCG/pull/345
* Magic fixes and new commands by @littlesera in https://github.com/littlesera/LSCG/pull/346
  * Fix height flickering on reduce/enlarge
  * Expose drug/netgun/spell keywords for MBS
  * Fix spell buttons when using beta room customization
  * Add commands to get/parse trimmed down outfit/polymorph codes straight to and from clipboard
    * /lscg get-outfit-code
    * /lscg get-polymorph-code
    * /lscg parse-code

**Full Changelog**: https://github.com/littlesera/LSCG/compare/v0.3.14...v0.3.15

<hr /><em>This discussion was created from the release <a href='https://github.com/littlesera/LSCG/releases/tag/v0.3.15'>v0.3.15</a>.</em>

**Comment by anchillabc** [comment]
I'm sorry to report that the flicker issue does not seem to be completely gone, just changed...

**Reply by littlesera** [reply]
tested with an assortment of blinking objects and didn't see it anymore after the change.. I was testing on the BC beta?
@anchillabc What different are you seeing?

**Reply by anchillabc** [reply]
I had tested on R96, not beta, though I also tested on R97 now and the results are the same.
What I had not noticed is that the blinking led issue is indeed taken care of, animations are the only remaining culprit.
I had bc responsive character talk on, which caused size flicker on mouth animation. Testing further, /anim jumpingjacks etc. also cause this.
Observing with an alt, manual pose changes also cause the same.

**Reply by littlesera** [reply]
fiddly fiddly fiddly 😂
Should have a new fix live on the dev branch if you want to test it out.
